<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyWidgetClass</class>
 <widget class="QWidget" name="CompanyWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompanyWidget</string>
  </property>
  <widget class="QListWidget" name="packagesListWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>190</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="recipientLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="recipientLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="streetLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="streetLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="numberLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="numberLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="xLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="xLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="yLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="yLineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
